docs/source/tutorials_source/platform/tutorial_active_learning.py

```diff
@@ -0,0 +1,190 @@
+#    token="yourToken" dataset_id="yourDatasetId"```
+
+
+
+
+# %%
+# Imports
+# -------
+#
+# Import the Python frameworks we need for this tutorial.
+
+import csv
+from typing import List, Dict, Tuple
+import numpy as np
+from sklearn.neighbors import KNeighborsClassifier
+
+from lightly.active_learning.agents.agent import ActiveLearningAgent
+from lightly.active_learning.config.sampler_config import SamplerConfig
+from lightly.active_learning.scorers.classification import ScorerClassification
+from lightly.api.api_workflow_client import ApiWorkflowClient
+from lightly.openapi_generated.swagger_client import SamplingMethod
+
+# %%
+# Definition of parameters
+path_to_embeddings_csv = "path/to/clothing-dataset-small/train/.../embeddings.csv"
+YOUR_TOKEN = "yourToken"  # your token of the web platform
+YOUR_DATASET_ID = "yourDatasetId"  # the id of your dataset on the web platform
+
+
+# %%
+# Definition of a dataset for the classifier based on the embeddings.csv
+# The kNN-classifier needs the embeddings as features for its classification. Thus we define a class to create
+# such a dataset out of the embeddings.csv. It also allows to choose only a subset of all samples
+# dependent on the filenames given.
+
+class CSVEmbeddingDataset:
+    def __init__(self, path_to_embeddings_csv: str):
+        with open(path_to_embeddings_csv, 'r') as f:
+            data = csv.reader(f)
+
+            rows = list(data)
+            header_row = rows[0]
+            rows_without_header = rows[1:]
+
+            index_filenames = header_row.index('filenames')
+            filenames = [row[index_filenames] for row in rows_without_header]
+
+            index_labels = header_row.index('labels')
+            labels = [row[index_labels] for row in rows_without_header]
+
+            embeddings = rows_without_header
+            indexes_to_delete = sorted([index_filenames, index_labels], reverse=True)
+            for embedding_row in embeddings:
+                for index_to_delete in indexes_to_delete:
+                    del embedding_row[index_to_delete]
+
+        # create the dataset as a dictionary mapping from the filename to a tuple of the embedding and the label
+        self.dataset: Dict[str, Tuple[np.ndarray, int]] = \
+            dict([(filename, (np.array(embedding_row, dtype=float), int(label)))
+                  for filename, embedding_row, label in zip(filenames, embeddings, labels)])
+
+    def get_features(self, filenames: List[str]) -> np.ndarray:
+        features_array = np.array([self.dataset[filename][0] for filename in filenames])
+        return features_array
+
+    def get_labels(self, filenames: List[str]) -> np.ndarray:
+        labels = np.array([self.dataset[filename][1] for filename in filenames])
+        return labels
+
+
+# %%
+# Upload the embeddings to the lightly web platform
+api_workflow_client = ApiWorkflowClient(token=YOUR_TOKEN, dataset_id=YOUR_DATASET_ID)
```

Comment:
I get an error here because of the missing host. I used a quick fix and just added the host argument. But we wanted to hide it!?
```
Traceback (most recent call last):
  File "tutorial_active_learning.py", line 128, in <module>
    api_workflow_client = ApiWorkflowClient(token=YOUR_TOKEN, dataset_id=YOUR_DATASET_ID)
TypeError: __init__() missing 1 required positional argument: 'host'
```

Reply:
Please use the quick fix now, as you want to set it to "https://api-dev.lightly.ai" anyways. I will restructure the host and its usage in all files in a later issue.